monero-wallet-rpc/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:20.04 AS install
+
+ARG VRS
+ENV VERSION=${VRS:-v0.17.2.3}
```

Comment:
if we're marking the version configurable, I'd say we _must_ make sha256 configurable too (otherwise it'd require changing the Dockerfile anyway - same for `monerod/Dockerfile`)

Reply:
Yes I think it is necessary to make the sha256 configurable too